src/common/textcmn.cpp

```diff
@@ -453,14 +476,15 @@ wxFont wxTextAttr::GetFont() const
 
     if (HasFontPixelSize())
     {
-        wxFont font(wxSize(0, fontSize), fontFamily, fontStyle, fontWeight, underlined, fontFaceName, encoding);
+        wxFont font(wxSize(0, wxFontInfo::ToIntPointSize(fontSize)), fontFamily, fontStyle, fontWeight, underlined, fontFaceName, encoding);
         if (strikethrough)
             font.SetStrikethrough(true);
         return font;
     }
     else
     {
         wxFont font(fontSize, fontFamily, fontStyle, fontWeight, underlined, fontFaceName, encoding);
+        font.SetFractionalPointSize(fontSize);
```

Comment:
It would be better to construct `wxFont` from `wxFontInfo`, which allows to specify the fractional font size directly, instead of creating it with the int font size and then setting it again.

Reply:
Just following the example a few lines up, at line 479, which was this before my changes.

wxFont font(wxSize(0, fontSize), fontFamily, fontStyle, fontWeight, underlined, fontFaceName, encoding);

Reply:
Besides, this hardly seems cleaner:

        wxFont font(wxFontInfo(fontSize).Family(fontFamily).Weight(wxFontBase::GetNumericWeightOf(fontWeight)).Underlined(underlined).FaceName(fontFaceName).Encoding(encoding));